f8a_report/helpers/report_helper.py

```diff
@@ -64,32 +64,34 @@ def __init__(self):
 
         self.emr_api = os.getenv('EMR_API', 'http://f8a-emr-deployment:6006')
 
+    def cleanup_tables(self, table_name, env_var, num_days):
```

Comment:
Keep the env out of this function. It should just take the table name and no.of days. The caller of the function has to ensure how to retrieve the day and pass it on to this.
```suggestion
    def cleanup_tables(self, table_name, num_days):
```